haystack/utils.py

```diff
@@ -28,6 +29,37 @@ def print_answers(results: dict, details: str = "all"):
         pp.pprint(results)
 
 
+def export_answers_to_csv(agg_results: list, output_file):
```

Comment:
Would it make sense to accept the output of `Finder.get_answers()` directly as a function parameter? In that case, we can do something like:

```python
pandas.DataFrame.from_dict(prediction["answers"]).to_csv()
```

Reply:
I think a better use case is when we aggregate mutliple Finder.get_answers() predictions and send it to this function.

And we also want the question in the CSV, so only using .from_dict(prediction["answers"]) wont have it included (but many quite useless parameters will be included). Maybe we can have a "detail" parameter as in the print function.